assets/script.js

```diff
@@ -0,0 +1,203 @@
+    $currentHorItem.append("<h6> Your lucky color is: " + horData.color + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky number is: " + horData.lucky_number + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky time is: " + horData.lucky_time + "</h6>" + "<br>");
+    console.log("Color is: ", color);
+    if (window.document.title !== "Astrozoders") {
+        tarot();
+    }
+    if (index) {
+
+        historySign = sign;
+        historyName = full_name;
+        console.log("type of historysing: ", typeof (historySign), "type of name: ", typeof (historyName));
+        historySign = JSON.stringify(historySign);
+        localStorage.setItem('sign', historySign);
+        historyName = JSON.stringify(historyName);
+        localStorage.setItem('name', historyName);
+
+    }
+}
+
+//  Function to build the URL for the API call
+function buildURL(sign) {
+
+    console.log("Inside buildURL");
+    if (index) {
+        console.log("index: ", index);
+        full_name = $("#name").val().trim();
+        console.log("Name input: ", full_name);
+        if (full_name === "") {
+            console.log("Name is not entered");
+            $('.modal').modal();
+            $('.modal').modal('open');
+            return;
+        }
+    }
+    clear();
+
+    console.log(sign);
+    var queryURL = "https://aztro.sameerkumar.website?";
+    var queryParams = {};
+    queryParams.sign = sign;
+    queryParams.day = "today";
+    var queryURL = queryURL + $.param(queryParams);
+    console.log("------------------------------------");
+    console.log("QueryURLall", queryURL);
+    console.log("'https://aztro.sameerkumar.website?sign=aries&day=today',");
+    // AjAX request to API to retrieve horoscope info
+    $.ajax({
+        type: 'POST',
+        url: 'https://aztro.sameerkumar.website?sign=aries&day=today',
+        url: queryURL,
+        success: function (horData) {
+            console.log(horData);
+            updatePage(horData, sign);
+        }
+    });
+};
+// this function will be executed when the page is loaded
+$(document).ready(function () {
+
+    $('.sidenav').sidenav();
+    console.log(window.document.title);
+    if (window.document.title === "Tarot") {
+        console.log("On Tarot Page");
+        tarot();
+    }
+    else if (window.document.title === "Astrozoders") {
+        index = true;
+        var name = JSON.parse(localStorage.getItem('name'));
+        console.log("On Main Page ", name);
+        if (name !== null) {
+            historySign = JSON.parse(localStorage.getItem('sign'));
+            historyName = JSON.parse(localStorage.getItem('name'))
+            $("#name").val(historyName);
+            console.log(historySign, " ", historyName);
+            sign = historySign;
+            buildURL(sign);
+
+        }
+    }
+    else {
+        console.log("On Sign Page");
+        var sign = window.document.title;
```

Comment:
You should have stored this information as a constant on the page rather than relying on `window.document.title`

`window.document.title` should be something that's for display purposes only, whereas a constant on the page could be for coding purposes